src/commands/apex/run/test.ts

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { CancellationTokenSource, TestLevel, TestResult, TestRunIdResult, TestService } from '@salesforce/apex-node';
+import {
+  Flags,
+  loglevel,
+  orgApiVersionFlagWithDeprecations,
+  requiredOrgFlagWithDeprecations,
+  SfCommand,
+  Ux,
+} from '@salesforce/sf-plugins-core';
+import { Messages, SfError } from '@salesforce/core';
+import { Duration } from '@salesforce/kit';
+
+import { RunResult, TestReporter } from '../../../reporters';
+import { resultFormat } from '../../../utils';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.load('@salesforce/plugin-apex', 'runtest', [
```

Comment:
general comment about this pattern of Messages.load (each individual message).

now that we do message linting, the bugs that this was meant to prevent are no longer able to be hit accidentally.  Also, load is still loading the whole file even if you specify some messages (found that out).  

So I'd recommend we stop naming them individuallly and just say 
```suggestion
const messages = Messages.load('@salesforce/plugin-apex', 'runtest');
```